README.md

```diff
@@ -37,6 +38,7 @@ Initialize helpers:
 ```go
 pack := occam.NewPack().WithVerbose()
 docker := occam.NewDocker()
+containerStructureTest := NewContainerStructureTest()
 ```
```

Comment:
Could you add an example of using the `containerStructureTest` to the Examples section?

Reply:
And in the example, maybe link out to the github repo for the project? Because it's lesser known in the project than Docker/Pack